events/userDebit.js

```diff
@@ -6,25 +6,53 @@
 
 module.exports = async (user, amount) => {
   try {
-    let userProfile = await user.client.db.get(`SELECT bastionCurrencies FROM profiles WHERE userID=${user.id}`);
+    let userModel = await user.client.database.models.guildMember.findOne({
+      attributes: [ 'bastionCurrencies' ],
+      where: {
+        userID: user.id
+      }
+    });
 
     /*
-    * If the user doesn't have a profile, create their profile
-    * & add Bastion Currencies.
-    */
-    if (!userProfile) {
-      return await user.client.db.run('INSERT INTO profiles (userID, bastionCurrencies) VALUES (?, ?)', [ user.id, parseInt(amount) ]);
+     * If the user doesn't have a profile, create their profile
+     * & add Bastion Currencies.
+     */
+    if (!userModel) {
+      return await user.client.database.models.guildMember.create({
+        userID: user.id,
+        guildID: 'ID', // TODO: Add support for guild ID.
```

Comment:
This is done temporarily to imitate global profiles (which was available up to v6). After this is merged, I'll implement changes to support per server profiles.